books/RayTracingInOneWeekend.html

```diff
@@ -1446,12 +1457,195 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
             return 0.5 * (rec.normal + color(1,1,1));
         }
+
+        vec3 unit_direction = unit_vector(r.direction());
+        auto a = 0.5*(unit_direction.y() + 1.0);
+        return (1.0-a)*color(1.0, 1.0, 1.0) + a*color(0.5, 0.7, 1.0);
+    }
     ...
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [main-with-interval]: <kbd>[main.cc]</kbd> The new main using interval]
 
 
 
+A New Camera Class
```

Comment:
The name of this chapter is confusing

It implies that it's replacing an already existing class.

You have `An Interval Class` above. Go with `A Camera Class` or, better, `Consolidate our Camera into a Class`

Reply:
Done: _Consolodating Camera Code Into Its Own Class_